src/main/java/com/alkemy/ong/model/Activity.java

```diff
@@ -0,0 +1,51 @@
+package com.alkemy.ong.model;
+
+import java.io.Serializable;
+import java.time.LocalDateTime;
+
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.Table;
+import javax.validation.constraints.NotNull;
+
+import org.hibernate.annotations.CreationTimestamp;
+import org.hibernate.annotations.SQLDelete;
+import org.hibernate.annotations.UpdateTimestamp;
+import org.hibernate.annotations.Where;
+
+import lombok.Data;
+
+@Entity
+@Data
+@Table(name = "activities")
+@SQLDelete(sql = "UPDATE activities SET deleted = true WHERE id=?")
+@Where(clause = "deleted=false")
+
+public class Activity implements Serializable{
+
+    private static final long serialVersionUID = 1L;
+    
+	@Id
+    @GeneratedValue (strategy = GenerationType.IDENTITY)
+    private Long id;
+	
+	@NotNull
```

Comment:
A todos los atributos que tenes como NOT NULL les falta agregar el mensaje de error correspondiente. Esto lo hacemos dentro de la misma annotation de la siguiente manera
@NotNull(message = "Name cannott be empty")

Reply:
Hola Turco , dale si buenísimo ahí agrego los mensajes de error.